### DISCUSSION TITLE
Check errors on build

### DISCUSSION THREAD

**Discussion by mitraecp** [nuxt/nuxt#17339] [Questions]
Hi guys! I would like to know how I enable error checks, such as syntax, so that these errors do not go during the build, or even [vue warn], I would like to break the build, so that the developer fixes it before sending a pull request

![image](https://user-images.githubusercontent.com/75841640/177331903-8aa30fa5-3721-41a2-9ed0-eae775a3fc35.png)
 here i have a purposeful error

![image](https://user-images.githubusercontent.com/75841640/177332155-16687b76-60b1-44a9-95e2-7a406da73aa9.png)
 but a simple warn, i want to expose this better...

and even with this simple error, the build occurs normally, and the app breaks in production =x

**Comment by Flanker72** [comment]
You can use [husky](https://www.npmjs.com/package/husky) with pre-commit / pre-push hook. It can run eslint checks and prevent committing/pushing invalid code. We use this approach in [Virto Commerce](https://github.com/VirtoCommerce/vc-theme-b2b-vue/blob/dev/.husky/pre-commit) to make code clean.
